test-workspace/dbt_project.yml

```
@@ -0,0 +1,21 @@
+test-paths: ["tests"]
+seed-paths: ["seeds"]
+macro-paths: ["macros"]
+snapshot-paths: ["snapshots"]
+
+target-path: "target"
+clean-targets:
+    - "target"
+    - "dbt_packages"
+
+models:
+  test_project:
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)